Makefile

```diff
@@ -378,6 +378,9 @@ e2e-tests: build
 	    exit 1; \
 	fi
 
+packaging-tests: cross-deb
+	$(E2E_RUN) test/e2e/packaging/cross-deb
```

Comment:
Shouldn't this be `packages/debian-10` ?

Reply:
> Shouldn't this be `packages/debian-10` ?

I mean `$(E2E_RUN) test/e2e/packages/debian-10` to be exact ?

Reply:
Definitely! Thanks! (Dangers of last minute renaming...)

Reply:
Fixed.